shared-data/python/opentrons_shared_data/liquid_classes/__init__.py

```diff
@@ -1 +1,25 @@
-"""opentrons_shared_data.liquid_classes: types and functions for accessing liquid class definitions."""
+"""Types and functions for accessing liquid class definitions."""
+import json
+
+from .. import load_shared_data
+from .liquid_class_definition import LiquidClassSchemaV1
+
+
+class LiquidClassDefinitionDoesNotExist(Exception):
+    """Specified liquid class' definition does not exist."""
```

Comment:
I don't think this has to be possessive.
```suggestion
    """Specified liquid class definition does not exist."""
```